pkg/cli/cvd.go

```diff
@@ -395,10 +400,18 @@ const (
 
 const (
 	AndroidBuildTopVarName   = "ANDROID_BUILD_TOP"
-	AndroidHostOutVarName    = "ANDROID_HOST_OUT"
 	AndroidProductOutVarName = "ANDROID_PRODUCT_OUT"
 )
 
+const (
```

Comment:
As these const variables are only used in one place I wouldn't create them and rather use the literals directly.

```
bin := filepath.Join(buildTop, "out/soong_ui")
cmdOut, err := exec.Command(bin, "--dumpvar-mode", "TARGET_ARCH").Output()
```

it makes it easier to read the command we are executing, otherwise we need to go back and forth. If you decide to keep the const variables, please declare them within the function making it closer to where it's used.